helm/airflow/values.yaml

```diff
@@ -151,7 +151,12 @@ images:
     pullPolicy: IfNotPresent
 
 # Environment variables for all airflow containers
-env: []
+env: 
+  - name: "NAMESPACE"
+    valueFrom:
+      fieldRef:
+        apiVersion: v1
+        fieldPath: metadata.namespace
```

Comment:
Do we want this here, or in the actual template that needs this env variable?

Reply:
My understanding is that the worker pods that run the DAGs also inherit from this same chart? Or do you know which template this would belong to?

The issue we were having was the 
```python
namespace = os.getenv('NAMESPACE')
``` 
call in our dags returning an empty string.